### PR TITLE
2269 update raspberry pi page

### LINKED ISSUE #2269: Screenly website URL is incorrect
Would you mind updating the URL to Screenly on the digital signage page
from screenlyapp.com to https://www.screenly.io?

---
*Reported from: https://www.ubuntu.com/internet-of-things/digital-signage*

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Fixup release variables and logic for raspberry pi buttons, thankyou and verification sections
- Add links to tutorials

## QA

- Download this branch
- ./run
- look at http://0.0.0.0:8001/download/raspberry-pi/
- select some images and test http://0.0.0.0:8001/download/raspberry-pi/thank-you and with some get parameters (from clicking a button)
- See that the downloads start, see that the verify instructions work
- Test the links on the page.

Fixes #2269 and #2300

### CODE REVIEW

File: templates/download/raspberry-pi/index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    </div>
+    <div class="col-3">
+      <p><a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=arm63+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 4 64-bit', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download 64-bit</a></p>
+      <p class="u-no-margin--bottom"><a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=armhf+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 4 32-bit', 'eventLabel' : '{{ releases.lts.short_version }}', 'eventValue' : undefined });">Download 32-bit</a></p>
```
Comment (anthonydillon) [inline templates/download/raspberry-pi/index.html]: Shouldn't these be downloading raspi4?

File: templates/download/raspberry-pi/thank-you.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Thank you for downloading Ubuntu Server for Raspberry Pi{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1RXsLbMB-LJVwiG0gAi0HgVEJvZ9_GIU3ESLJ6KmBb54{% endblock meta_copydoc %}
+{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+
+{% block content %}
+<meta http-equiv="refresh" content="3;url=http://cdimage.ubuntu.com/releases/{{ version }}/release/ubuntu-{{ version }}-preinstalled-server-{{ architecture }}.img.xz">
```
Comment (anthonydillon) [inline templates/download/raspberry-pi/thank-you.html]: Just double checking the preinstalled-server is the correct download here?

**Comment by anthonydillon** [comment]
QA: Small screens
- Hide the image in the hero.
- Hide the Raspbarry pi image section
- Add what version of Raspberry Pi the download is for in the download buttons but hide them on larger screens.

File: templates/download/raspberry-pi/index.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    <div class="col-6">
+      <h2>Which image should I pick?</h2>
+      <p>If you are wondering which download to pick, here are a few pointers on the options.</p>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+        <h3>Ubuntu {{ releases.lts.full_version }} LTS vs Ubuntu {{ releases.latest.full_version }}</h3>
+      <hr class="u-sv1">
+      <p>Generally we recommend the Long term support (LTS) releases as they are supported for five years of free security and maintenance updates, guaranteed.  However, the interim releases often have the newest features and hardware support. Ubuntu {{ releases.latest.full_version }} comes with nine months of support, until , of support so there is really no bad choice.</p>
```
Comment (anthonydillon) [inline templates/download/raspberry-pi/index.html]: This paragraph doesn't make much sense.
- "supported for five years _of_ free security and maintenance updates" seems the word of it not correct or makes the sentence difficult to read.
- the latest release EOL date is missing
- is "Long team support" capitalised?

File: templates/download/raspberry-pi/index.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      <h1>Install Ubuntu Server on a Raspberry Pi 2, 3 or 4</h1>
+      <p>Running Ubuntu Server on your Raspberry Pi is easy. Just pick the OS image you want, flash it onto a microSD card, load it onto your pi and away you go.</p>
+    </div>
+    <div class="col-4 u-vertically-center">
+      <img src="https://assets.ubuntu.com/v1/31bd2627-logo-raspberry-pi.svg" width="100" alt="Raspberry Pi">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered" style="padding-bottom: 1.5rem;">
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/download/raspberry-pi/index.html]: I added  u-sv3 to the row instead

File: templates/download/raspberry-pi/index.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-3">
+      <h2 class="u-no-margin--bottom">Download your Ubuntu Pi image</h2>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/download/raspberry-pi/index.html]: Do you mean the `u-equal-height` isn't needed?

File: templates/download/raspberry-pi/index.html [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+{% block content %}
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-8">
+      <h1>Install Ubuntu Server on a Raspberry Pi 2, 3 or 4</h1>
+      <p>Running Ubuntu Server on your Raspberry Pi is easy. Just pick the OS image you want, flash it onto a microSD card, load it onto your pi and away you go.</p>
+    </div>
+    <div class="col-4 u-vertically-center">
+      <img src="https://assets.ubuntu.com/v1/31bd2627-logo-raspberry-pi.svg" width="100" alt="Raspberry Pi">
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/download/raspberry-pi/index.html]: Yes

File: templates/download/raspberry-pi/index.html [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  </div>
+</section>
+
+<section class="p-strip is-bordered" style="padding-bottom: 1.5rem;">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-3">
+      <h2 class="u-no-margin--bottom">Download your Ubuntu Pi image</h2>
+    </div>
+    <div class="col-3">
+      <div><img src="https://assets.ubuntu.com/v1/8542c958-Raspberry+Pi+2.png" alt=""></div>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/download/raspberry-pi/index.html]: Done.

File: templates/download/raspberry-pi/index.html (hunk 1 above)
Comment (pmahnke) [reply templates/download/raspberry-pi/index.html]: That does not currently exist as a separate image.

File: templates/download/raspberry-pi/thank-you.html (hunk 2 above)
Comment (pmahnke) [reply templates/download/raspberry-pi/thank-you.html]: Yes.  These might change tonight.  But they are preinstalled, ready to be flashed onto a microSD card.

File: templates/download/shared/_verify-checksums.html [hunk 8]
```diff
@@ -7,7 +7,7 @@
         <small>Run this command in your terminal in the directory the iso was downloaded to verify the SHA256 checksum:</small><br />
         <code style="display: block; margin-top: 1rem;">echo "{{ releases.checksums[system][version] }}" | shasum -a 256 --check</code><br />
         <small>You should get the following output:</small><br />
-        <code style="display: block; margin-top: 1rem;">ubuntu-{{ version }}-{{ system }}-{{ architecture }}.iso: OK</code><br />
+        <code style="display: block; margin-top: 1rem;">ubuntu-{{ version }}-{% if system != architecture %}{{ system }}-{% else %}preinstalled-server-{% endif %}{{ architecture }}.iso: OK</code><br />
```
Comment (anthonydillon) [inline templates/download/shared/_verify-checksums.html]: Conditionally check if system and version exist before referencing the checksums

**Pull request merged**